public/new-image-card.html

```diff
@@ -163,20 +170,19 @@
           this.$.styleFileInput.click();
         },
         contentFileChanged_: function() {
-          this.handleFileInput_('contentFile_', this.$.contentFileInput.files, this.$.contentImagePreview);
+          this.handleFileInput_('contentFiles_', this.$.contentFileInput.files, this.$.contentImagePreview);
         },
         styleFileChanged_: function() {
-          this.handleFileInput_('styleFile_', this.$.styleFileInput.files, this.$.styleImagePreview);
+          this.handleFileInput_('styleFiles_', this.$.styleFileInput.files, this.$.styleImagePreview);
         },
         handleFileInput_: function(fileProperty, files, imageElement) {
-          this.set(fileProperty, null);
-          if (files.length != 1) {
-            return;
-          }
-          if (!files[0].type.startsWith('image/')) {
-            return;
+          this.set(fileProperty, []);
```

Comment:
check that files.length > 0? Not sure if it's possible for handleFileInput_ to be called with files.length == 0, but if it was possible, then readAsDataURL below would fail.